theme/brb/style.scss

```diff
@@ -59,11 +61,22 @@ body {
             }
 
             @media (max-width: 991px) {
-                .cloud-logo {
-                    width: 100% !important;
-                    margin-left: 0 !important;
+               .cloud-logo {
+                    background: $logoBackgroundTablet;
+                    margin-left: 0.5em!important;
+                    position: absolute;
+                    left: 0;
+                    width: 13rem !important;
```

Comment:
differentiate between parent and root element in styling one specific element does not sound like a good idea